screen/budgets/src/screenshotTestDebug/kotlin/BudgetScreenPreviewTest.kt

```diff
@@ -0,0 +1,15 @@
+import androidx.compose.runtime.Composable
+import com.ivy.budgets.BudgetScreenUiTest
+import com.ivy.ui.testing.IvyPreviews
+
+@IvyPreviews
+@Composable
+private fun PreviewBudgetScreenLight() {
```

Comment:
Can't we use the previews from the main source-set?

Reply:
yep we can, but the purpose of it is the same as the comment below in IvyPreviews annotation